epimodel/exports/epidemics_org.py

```diff
@@ -59,21 +70,73 @@ def write(self, path, name=None):
             with open(exdir / fname, "wt") as f:
                 json.dump(er.data_ext, f)
         with open(exdir / MAIN_DATA_FILENAME, "wt") as f:
-            json.dump(self.to_json(), f, indent=2, default=types_to_json)
+            json.dump(self.to_json(), f, default=types_to_json)
```

Comment:
This reduces the json from 1.4MB to 680KB

Reply:
Yeah, that was debug only.